tools/ossf_best_practices/gold_criteria.md

```diff
@@ -0,0 +1,248 @@
+- [Team Discussion](https://github.com/nodejs/security-wg/pull/956#discussion_r1307405014)
+
+> The project MUST have FLOSS automated test suite(s) that provide at least 80% branch coverage if there is at least one FLOSS tool that can measure this criterion in the selected language.
+
+**Met. This is part of the CI Checks in place**
+
+Context:
+- [CII Best Practices: Quality](https://github.com/coreinfrastructure/best-practices-badge/blob/main/docs/other.md#upgrade-quality-1)
+- [CII Best Practices: Test Branch Coverage 80%](https://github.com/coreinfrastructure/best-practices-badge/blob/main/docs/other.md#test_branch_coverage80)
+- [Team Discussion](https://github.com/nodejs/security-wg/pull/956#discussion_r1307405888)
+
+
+# Security
+
+## Use basic good cryptographic practices
+
+_Note that some software does not need to use cryptographic mechanisms. If your project produces software that (1) includes, activates, or enables encryption functionality, and (2) might be released from the United States (US) to outside the US or to a non-US-citizen, you may be legally required to take a few extra steps. Typically this just involves sending an email. For more information, see the encryption section of [Understanding Open Source Technology & US Export Controls](https://www.linuxfoundation.org/resources/publications/understanding-us-export-controls-with-os-projects/)._
+
+> The software produced by the project MUST support secure protocols for all of its network communications, such as SSHv2 or later, TLS1.2 or later (HTTPS), IPsec, SFTP, and SNMPv3. Insecure protocols such as FTP, HTTP, telnet, SSLv3 or earlier, and SSHv1 MUST be disabled by default, and only enabled if the user specifically configures it. If the software produced by the project does not support network communications, select "not applicable" (N/A).
+
+**N/A**
+
+Context:
+- [CII Best Practices: Security](https://github.com/coreinfrastructure/best-practices-badge/blob/main/docs/other.md#upgrade-security-1)
+- [Team Discussion](https://github.com/nodejs/security-wg/pull/956#discussion_r1307415866)
+- [See related question in Silver Criteria](/tools/ossf_best_practices/silver_criteria.md)
+
+> The software produced by the project MUST, if it supports or uses TLS, support at least TLS version 1.2. Note that the predecessor of TLS was called SSL. If the software does not use TLS, select "not applicable" (N/A).
+
+**Met**
+
+Context:
+- [CII Best Practices: Security](https://github.com/coreinfrastructure/best-practices-badge/blob/main/docs/other.md#upgrade-security-1)
+- [Team Discussion](https://github.com/nodejs/security-wg/pull/956#discussion_r1307415066)
+- [See related question in Silver Criteria](/tools/ossf_best_practices/silver_criteria.md)
+
+
+## Secured delivery against man-in-the-middle (MITM) attacks
+
+> The project website, repository (if accessible via the web), and download site (if separate) MUST include key hardening headers with nonpermissive values. (URL required)
```

Comment:
We need to understand it

Reply:
I created this issue to follow up with the discussion. https://github.com/nodejs/security-wg/issues/1190